release-team/release-team-removal.md

```diff
@@ -0,0 +1,48 @@
+# Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emiritus Advisor to find a replacement if they would like to step down.
+
+# Removal Process
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affect the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances. 
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by EA (we host 2-3 sessions in different timezone)
+- Failure to attend shadow orientation hosted by role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
+- Failure to attend 3 consecutive release meeting without communication to role lead
```

Comment:
So to clarify, missing consecutive release meetings is fine as long as it's been cleared by the role lead, correct? The different time zones make it a little tricky for some of the shadows to go to all of the release team meetings so it would be good to add this to the shadow application process. It looks like it's missing from the[ release shadows doc](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md) and we could potentially add a point to the [logistics section](https://github.com/kubernetes/sig-release/blob/master/release-team/release-team-selection.md#logistics) in the release team selection as well.